lively.ide/world.js

```diff
@@ -897,7 +897,8 @@ export class LivelyWorld extends World {
     if (target._responsiveHalo) target._responsiveHalo.remove();
     if (!Halo) return;
     if (!obj.isArray(target) && !target.halosEnabled) return;
-    if (this.halos().filter(halo => halo.target === target).length > 0) return;
```

Comment:
I am confused - how were we ever able to show a new halo? :O

Reply:
Well, it only considered halos for a particular target? I assume this expression was `false` most of the time.

Reply:
Nevermind me, I read that as not greater than 0 :upside_down_face: